DigitalLearningSolutions.Data/Services/UserService.cs

```diff
@@ -242,9 +259,77 @@ public IEnumerable<DelegateUserCard> GetDelegateUserCardsForWelcomeEmail(int cen
             );
         }
 
+        public void UpdateAdminUserPermissions(
+            int adminId,
+            bool isCentreAdmin,
+            bool isSupervisor,
+            bool isTrainer,
+            bool isContentCreator,
+            bool isContentManager,
+            bool importOnly,
+            int categoryId

```

Comment:
Could we wrap this in a model - e.g. `AdminRoles`. Then you can also do something like EditRolesViewModel.GetAdminRoles() in the controller when calling this method